apps/condo/domains/user/schema/User.js

```diff
@@ -374,6 +385,23 @@ const User = new GQLListSchema('User', {
             ) {
                 await updateEmployeesRelatedToUser(context, updatedItem)
             }
+
+            if (updatedItem.type === STAFF) {
+                const isEmailChanged = updatedItem.email !== (existingItem ? existingItem.email : null)
```

Comment:
Why you need `(existingItem ? existingItem.email : null)` here? Why not just ... `!== existingItem?.email`?